src/System.Private.CoreLib/shared/System/Diagnostics/Tracing/EventSource.cs

```diff
@@ -3913,19 +3896,6 @@ private bool SelfDescribingEvents
                                 ((m_config & EventSourceSettings.EtwSelfDescribingEventFormat) != 0));
                 return (m_config & EventSourceSettings.EtwSelfDescribingEventFormat) != 0;
             }
-            set
```

Comment:
@noahfalk @brianrob These two properties were private methods, reachable via private reflection only. Just want to make sure - are you aware of any folks using these properties via private reflection?

Reply:
I am unaware of any private reflection usage on these. I think @brianrob is on vacation at the moment so we probably won't hear from him.